src/Core/Domain/OrderReturn/QueryResult/OrderDetailCustomization.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+namespace PrestaShop\PrestaShop\Core\Domain\OrderReturn\QueryResult;
+
+use PrestaShop\PrestaShop\Adapter\Entity\Product;
+
+/**
+ * @todo duplicate of PrestaShop\PrestaShop\Core\Domain\Order\QueryResult\OrderProductCustomizationForViewing
+ * May be merged later but need to be sure that OrderReturnOrderDetail and OrderDetail are the same thing
+ */
+class OrderDetailCustomization
+{
+    /**
+     * @var int
+     */
+    private $type;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $value;
+
+    /**
+     * @var string
+     */
+    private $image;
+
+    /**
+     * @param int $type
+     * @param string $name
+     * @param string $value
+     */
+    public function __construct(int $type, string $name, string $value)
+    {
+        $this->type = $type;
+        $this->name = $name;
+        $this->value = $value;
+        if (Product::CUSTOMIZE_FILE === $this->type) {
+            $this->image = _THEME_PROD_PIC_DIR_ . $this->value . '_small';
```

Comment:
Please don't use a constant here. This check should be done outside of this PR.

Reply:
Changed to use OrderDetailCustomizationForViewing after close look